config/schemas/config.yaml

```diff
@@ -0,0 +1,2854 @@
+        additionalProperties:
+          type: boolean
+        type: object
+    type: object
+  falco:
+    properties:
+      affinity:
+        $ref: '#/$defs/affinity'
+      alerts:
+        properties:
+          enabled:
+            default: false
+            type: boolean
+          hostPort:
+            default: http://alertmanager-operated.monitoring:9093
+            format: uri
+            type: string
+          priority:
+            default: notice
+            type: string
+          type:
+            default: alertmanager
+            type: string
+        type: object
+      customRules:
+        type: object
+        additionalProperties:
+          type: string
+          format: yaml
+      driver:
+        properties:
+          ebpf:
+            properties:
+              hostNetwork:
+                default: false
+                type: boolean
+              path:
+                type: string
+            type: object
+          kind:
+            default: module
+            type: string
+        type: object
+      enabled:
+        default: true
+        type: boolean
+      falcoExporter:
+        properties:
+          affinity:
+            $ref: '#/$defs/affinity'
+          nodeSelector:
+            $ref: '#/$defs/nodeSelector'
+          resources:
+            $ref: '#/$defs/resources'
+          tolerations:
+            $ref: '#/$defs/tolerations'
+        type: object
+      falcoSidekick:
+        properties:
+          affinity:
+            $ref: '#/$defs/affinity'
+          nodeSelector:
+            $ref: '#/$defs/nodeSelector'
+          resources:
+            $ref: '#/$defs/resources'
+          tolerations:
+            $ref: '#/$defs/tolerations'
+        type: object
+      nodeSelector:
+        $ref: '#/$defs/nodeSelector'
+      resources:
+        $ref: '#/$defs/resources'
+      rulesFiles:
+        properties:
+          default:
+            properties:
+              enabled:
+                default: true
+                type: boolean
+              version:
+                default: 2.0.0
+                type: string
+            type: object
+          incubating:
+            properties:
+              enabled:
+                default: false
+                type: boolean
+              version:
+                default: 2.0.0
+                type: string
+            type: object
+          sandbox:
+            properties:
+              enabled:
+                default: false
+                type: boolean
+              version:
+                default: 2.0.0
+                type: string
+            type: object
+        type: object
+      tolerations:
+        $ref: '#/$defs/tolerations'
+      tty:
+        default: true
+        type: boolean
+    type: object
+  fluentd:
+    properties:
+      aggregator:
+        properties:
+          affinity:
+            $ref: '#/$defs/affinity'
+          buffer:
+            properties:
+              chunkLimitSize:
```

Comment:
This one should either be a generic object or include all parameters from [here](https://docs.fluentd.org/configuration/buffer-section#parameters). (Converted from snake case to camel case.)